File: ScpProject/StrengthConApp/src/app/Models/Exercise.ts
import { ITaggable } from './../Interfaces/ITaggable';
import { Tag } from './Tag';
export class Exercise implements ITaggable {
    Id: number;
    Notes: string;
    Name: string;
    CreatedUserId: number;
    IsDeleted: boolean;
    Percent: number;
    PercentMetricCalculationId: number;
    Tags: Tag[] = [];
    CanModify: boolean = true;
    VideoURL: string = '';
    CalcMetricName: string = '';
};

File: ScpProject/BL/BusinessObjects/Exercise.cs
using System.Collections.Generic;
using b = BL.BusinessObjects;

namespace BL.BusinessObjects
{
    public class Exercise
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Notes { get; set; }
        public int CreatedUserId { get; set; }
        public bool IsDeleted { get; set; }
        public double? Percent { get; set; }
        public int? PercentMetricCalculationId { get; set; }
        public List<b.Tag> Tags { get; set; }
        public bool CanModify { get; set; }
        public string VideoURL { get; set; }
        public string CalcMetricName { get; set; }
    }
}
